firebase_admin/db.py

```diff
@@ -544,10 +542,16 @@ class _Client(object):
     marshalling and unmarshalling of JSON data.
     """
```

Comment:
You should document what the arguments are now.  The signature no longer conveys this information